circle.yml

```diff
@@ -19,6 +19,9 @@ checkout:
     - mv ${HOME}/flux ${GOPATH}/src/github.com/weaveworks/
     - ln -s ${GOPATH}/src/github.com/weaveworks/flux ${HOME}/flux # Circle needs this to be here, apparently
     - cd ${GOPATH}/src/github.com/weaveworks/flux
+    - sudo pip install pyyaml
+    - git config --global user.email "example@example.com"
+    - git config --global user.name "Weave Flux test user"
```

Comment:
Git will refuse to push if you don't have them (yes you can probably supply them in the invocation, but this is easier)